prow/resultstore/files_test.go

```diff
@@ -112,11 +100,13 @@ func (i *fakeIterator) Next(_ context.Context) (pio.ObjectAttributes, error) {
 	}
 	oa.Name = strings.TrimPrefix(n, i.bucket)
 	oa.IsDir = strings.HasSuffix(n, "/")
+	oa.Size = i.finder.files[n].Size
 	return oa, nil
 }
 
 func TestArtifactFiles(t *testing.T) {
 	ctx := context.Background()
+	mime.AddExtensionType(".log", "text/plain")
```

Comment:
Nit: I think this can be removed since it is in the init block now. That would also ensure the init block content is working as intended.

Reply:
Oh, good catch! I was checking how that worked, and meant to remove it. I dislike the nature of that package having  a global side effect. (Would be better if mime had the ability to get a private resolver and customize it. Alas.)